cluster/gce/gci/configure-helper.sh

```diff
@@ -2609,4 +2609,4 @@ if [[ "$#" -eq 1 && "${1}" == "--source-only" ]]; then
    :
 else
    main "${@}"
-fi
+fi
```

Comment:
I don't think you meant to change this

Reply:
Yes, and I've reverted this change already. Perhaps, you were reviewing commits instead of a full changes.